src/app/modules/home/main/main.component.css

```diff
@@ -0,0 +1 @@
+/* No CSS *//*# sourceMappingURL=main.component.css.map */
```

Comment:
Is this file necessary since it's empty and there is a .scss?

Reply:
I think this is from a scss plugin in my IDE. I removed it

Reply:
I think it came from a SCSS plugin in my IDE. I removed it